babel/src/board.js

```diff
@@ -1,214 +1,13 @@
 'use strict';
 import * as _ from 'lodash';
+import iterchunks from './tools/iterchunks';
+import evaluate as affectOrbs.evaluate from './affectOrbs/evaluate';
+import unmatch as affectOrbs.unmatch from './affectOrbs/unmatch';
+// import './affectOrbs';
```

Comment:
I'm not sure I understand completely, but I think this might be what you want.

```
babel/src/board.js
babel/src/orbs/evaluate.js
babel/src/orbs/unmatch.js
```

It looks like you have identical code in `babel/src/affectOrbs.js` and `babel/src/affectOrbs/*.js`.